apps/web/app/layout.tsx

```diff
@@ -1,22 +1,110 @@
-import "./globals.css";
-import type { Metadata } from "next";
-import { Inter } from "next/font/google";
+"use client";
```

Comment:
explore more about "use client"; and use it approriately in e.g page level or detailed component level.